tcf_website/static/reviews/review_form.js

```diff
@@ -0,0 +1,132 @@
+/*
+ * Cascading dropdown data population for the "new review" form
+ * in review.py and reviews/new.html
+ * Author: j-alicia-long, 11/22/2020
+*/
+
+// Executed when DOM is ready
+jQuery(function($) {
+    // Clear & disable sequenced dropdowns
+    clearDropdown("#subject");
+    clearDropdown("#courseID");
+    clearDropdown("#instructor");
+    clearDropdown("#semester");
+    // Enable subject selector, disable the following
+    $("#subject").prop("disabled", false);
+    $("#courseID").prop("disabled", true);
+    $("#instructor").prop("disabled", true);
+    $("#semester").prop("disabled", true);
+
+    // Fetch all subdepartment data from API
+    var subdeptEndpoint = "http://localhost:8000/api/subdepartments/";
```

Comment:
You don't need `http://localhost:8000` in the beginning in both local and production environments. It can start with `/api`. Change all occurrences of this.

Reply:
oh pffft yes I def need to change that, thanks for the reminder!

Reply:
@Bartleby2718 do you know how this might change if we decide to have a private api (i.e. require tokens for access)? I'd assume that the production environment should be able to access the API it's hosting by itself without further setup, but I don't know.